### PR TITLE
Add support for tfsec Terraform linter

### PR DESCRIPTION
**Pull request by koka831**
This PR addresses https://github.com/dense-analysis/ale/issues/4229

### CODE REVIEW

**Review by hsanson — APPROVED** [review]
Looks good, thanks.

File: doc/ale-terraform.txt [hunk 1]
```diff
@@ -114,6 +114,25 @@ g:ale_terraform_tflint_options                 *g:ale_terraform_tflint_options*
...
+g:ale_terraform_tfsec_executable             *g:ale_terraform_tfsec_executable*
+                                             *b:ale_terraform_tfsec_executable*
+
+  Type: |String|
+  Default: `'tfsec'`
+
+  This variable can be changed to use a different executable for tfsec.
+
+g:ale_terraform_tfsec_options                *g:ale_terraform_tfsec_executable*
+                                             *b:ale_terraform_tfsec_executable*
```
Comment (benknoble) [inline doc/ale-terraform.txt:129]: Assuming the tag was a typo for `[gb]:ale_terraform_tfsec_options`? This causes `:helptags` to error

Comment (koka831) [reply doc/ale-terraform.txt:129]: oh, I'll fix it.

Comment (benknoble) [reply doc/ale-terraform.txt:129]: @hsanson perhaps it's worth adding something like `:helptags ALL` to the test suite?

Comment (hsanson) [reply doc/ale-terraform.txt:129]: https://github.com/dense-analysis/ale/issues/4328

**Pull request merged**